backup_and_restore/application_backup_and_restore/backing_up_and_restoring/oadp-scheduling-backups-doc.adoc

```diff
@@ -57,14 +57,14 @@ spec:
     includedNamespaces:
     - <namespace> <2>
     storageLocation: <velero-sample-1> <3>
-    defaultVolumesToRestic: true <4>
+    defaultVolumesToFsBackup: true <4>
     ttl: 720h0m0s
 EOF
 ----
 <1> `cron` expression to schedule the backup, for example, `0 7 * * *` to perform a backup every day at 7:00.
 <2> Array of namespaces to back up.
 <3> Name of the `backupStorageLocations` CR.
-<4> Optional: Add the `defaultVolumesToRestic: true` key-value pair if you are backing up volumes with Restic.
+<4> Optional:  OADP 1.2 and later: Add the `defaultVolumesToFsBackup: true` key-value pair if you are backing up volumes with Restic. OADP 1.1: Add the `defaultVolumesToRestic: true` key-value pair if you are backing up volumes with Restic.
```

Comment:
It is probably the best way to handle it. It is not ideal, but it is what it is, so it is